example/lib/collection.js

```diff
@@ -0,0 +1,29 @@
+var mongodb = require('mongodb');
```

Comment:
@wtrocki might be nice to plug @MikeyBurkman's mongo module here to make the code a little cleaner? https://github.com/MikeyBurkman/rhmap-mongodb